pyOCD/tools/gdb_server.py

```diff
@@ -44,6 +44,10 @@ def __init__(self):
         self.args = None
         self.gdb_server_settings = None
         self.echo_msg = None
+        self.input_args = None
```

Comment:
What's the intention here with input_args? The only use I see it's in the test script. How this differ from self.args